src/node.cc

```diff
@@ -3265,7 +3273,8 @@ static void ParseArgs(int* argc,
   new_argv[0] = argv[0];
 
   unsigned int index = 1;
-  while (index < nargs && argv[index][0] == '-') {
+  bool short_circuit = false;
```

Comment:
This was the easiest way I could find to pass all args after --process-prof to the processing script. Is there a more idiomatic way to accomplish this?